docs/osquery/osquery.asciidoc

```diff
@@ -120,18 +121,76 @@ image::images/scheduled-pack.png[Shows queries in the pack and details about eac
 . View scheduled query results in <<discover,*Discover*>> or the drag-and-drop <<lens,*Lens*>> editor.
 
 [float]
-[[osquery-prebuilt-packs]]
-== Prebuilt Elastic packs
+[[osquery-manage-query]]
+== Save queries
+
+You can save queries in two ways:
+
+* After running a live query, click the *Save for later* link.
+* From the *Saved queries* tab, click the **Add saved query** button.
+
+Once you save a query, you can only edit it from the *Saved queries* tab.
+
+To add or edit saved queries from the *Saved queries* tab:
```

Comment:
I don't think this sentence is necessary.  I'd remove and put a colon after the previous sentence:

> Once you save a query, you can only edit it from the *Saved queries* tab:

Reply:
Good catch, will update.